tests/test_faster_rcnn.py

```diff
@@ -312,28 +303,22 @@ def test_faster_rcnn_load(mock_create, config, dataset):
     assert os.listdir(log_dir)[0].startswith("events.out.tfevents")
 
 
-@patch("datasetinsights.estimators.faster_rcnn.Dataset.create")
-def test_create_dataset(mock_create, config, dataset):
+def test_len_dataset(config, dataset):
     """test download data."""
-    mock_create.return_value = dataset
-    train_dataset = create_dataset(config, TRAIN)
-    assert len(dataset.images) == len(train_dataset)
+    assert len(dataset.images) == len(dataset)
 
 
-@patch("datasetinsights.estimators.faster_rcnn.Dataset.create")
-def test_create_dryrun_dataset(mock_create, config, dataset):
+def test_create_dryrun_dataset(config, dataset):
     """test create dryrun dataset."""
-    mock_create.return_value = dataset
-    train_dataset = create_dataset(config, TRAIN)
+    train_dataset = dataset
     train_dataset = create_dryrun_dataset(config, train_dataset, TRAIN)
     assert config.train.batch_size * 2 == len(train_dataset)
 
 
-@patch("datasetinsights.estimators.faster_rcnn.Dataset.create")
-def test_create_sampler(mock_create, config, dataset):
+def test_create_sampler(config, dataset):
     """test create sampler."""
-    mock_create.return_value = dataset
-    train_dataset = create_dataset(config, TRAIN)
```

Comment:
why are we creating local reference train_dataset = dataset, why can't we use dataset directly?

Reply:
There is no good reason, i can change this if its more readable